### ISSUE TITLE
X3F_extract unable to extract files from APS-C sized Sigma Quattro H files

### ISSUE THREAD

**Issue by alextutubalin** [Kalpanika/x3f#107]
When trying to use x3f_extract on APS-C size images (both full-res and half-res), this utility displays 'Could not find any matching RAW format' messages.
APS-H sized files from same camera are OK.

Here is archive, containing all 4 possible sized X3F files from this camera: https://www.dropbox.com/s/2z75tghs35slu9w/SQuattroHAllSizes.zip?dl=0

**Comment by rolkar** [comment]
Thanx for the report and the links to the files. I shall take a look. I do not promise and time frame for fixing it. The work on X3F Tools is on a very low pace.

Sigma is very problematic with their file format. Most is ad hoc and it is not always trivial to fix things

Sometimes it is just to add the ID for the file format. And sometimes not.

In this particular case we have those AF points. They are in a regular grid, that should be the same for APS-C. But, we need to find the offset. It is, unfortunately, hard coded for sd Q in our code. We have not found where they are defined in the meta data.

**Comment by alextutubalin** [comment]
The files contains mostly blue sky, so AF points are clearly visible (if AF intepolation is turned off)

But the problem is not with AF points, but with data extraction per see: DNG file is not created for APS-C files (both high- and low- res)

**Comment by rolkar** [comment]
Yes, I understood. But, to be able to make correct DNG files we need to interpolate the AF points.
